modules/editor/inserter/index.js

```diff
@@ -0,0 +1,22 @@
+const h = wp.element.createElement;
```

Comment:
Right, but I don't think it makes a lot of sense in our context—why not `element` or `node`?